components/card/article/src/index.js

```diff
@@ -33,26 +33,32 @@ export default function CardArticle (props) {
     description,
     tag,
     comments,
-    lazyLoad
+    lazyLoad,
+    featured = false
   } = props
   const Link = props.linkFactory
```

Comment:
We can move this assignment into the previous object and then move the whole destructured object to the function definition, so that we get rid off the first 2 consts.